.github/workflows/preview.yml

```
@@ -51,29 +51,12 @@ jobs:
           tar -xf shimmer.tar
           tar -xf next.tar
       # Convert to Vercel project and upload
-      - name: Install Vercel CLI
-        run: npm install --global vercel@latest
-      - name: Pull Vercel Environment Information
-        run: vercel pull --yes --environment=preview --token=${{ secrets.VERCEL_TOKEN }}
-      - name: Build Project Artifacts
-        run: vercel build --token=${{ secrets.VERCEL_TOKEN }}
-      - name: Deploy Project Artifacts to Vercel
-        run: vercel deploy --prebuilt --token=${{ secrets.VERCEL_TOKEN }} > vercel_output.txt
-      # Set the deploy URL as an output
-      - name: Set deploy url
-        id: deploy_url
-        run: echo "DEPLOY_URL=$(cat vercel_output.txt | awk 'END{print}')" >> $GITHUB_OUTPUT
-      # Create a comment on the pull request with the deployed URL
-      - name: Comment on pull request
-        uses: actions/github-script@v6.4.0
+      - name: Deploy to Vercel Action
+        uses: BetaHuhn/deploy-to-vercel-action@v1
         with:
-          github-token: ${{ secrets.GITHUB_TOKEN }}
-          script: |
-            const DEPLOY_URL = '${{ steps.deploy_url.outputs.DEPLOY_URL }}'
-            const COMMIT_SHA = '${{ github.event.pull_request.head.sha }}'
-            github.rest.issues.createComment({
-              issue_number: context.issue.number,
-              owner: context.repo.owner,
-              repo: context.repo.repo,
-              body: `This pull request has been deployed to Vercel.\n\n\nLatest commit: ${COMMIT_SHA}\n\n:white_check_mark: Preview: ${DEPLOY_URL}`
-            })
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          VERCEL_TOKEN: ${{ secrets.VERCEL_TOKEN }}
+          VERCEL_ORG_ID: ${{ secrets.VERCEL_ORG_ID }}
+          VERCEL_PROJECT_ID: ${{ secrets.VERCEL_PROJECT_ID }}
+          PR_LABELS: false
+          PREBUILT: true
```

It would be good to add `PRODUCTION: false` here just in case.
Also, If you want the deploys to show in the `Preview` environment, then you should add `GITHUB_DEPLOYMENT_ENV: Preview`. We already had problems because we didn't add this line :sweat_smile:

---

Not familiar enough with Github and Action to tell what exactly that does 😅 
But I can add it 👍 
```suggestion
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
          VERCEL_TOKEN: ${{ secrets.VERCEL_TOKEN }}
          VERCEL_ORG_ID: ${{ secrets.VERCEL_ORG_ID }}
          VERCEL_PROJECT_ID: ${{ secrets.VERCEL_PROJECT_ID }}
          PR_LABELS: false
          PREBUILT: true
          PRODUCTION: false
          GITHUB_DEPLOYMENT_ENV: Preview
```